cardano-crypto-tests/src/Bench/Crypto/DSIGN.hs

```diff
@@ -0,0 +1,80 @@
+{-#LANGUAGE TypeApplications #-}
+{-#LANGUAGE ScopedTypeVariables #-}
+{-#LANGUAGE TypeFamilies #-}
+{-#LANGUAGE FlexibleContexts #-}
+{-#LANGUAGE MultiParamTypeClasses #-}
+{-#LANGUAGE FunctionalDependencies #-}
+module Bench.Crypto.DSIGN
+  ( benchmarks
+  ) where
+
+import Data.Proxy
+import Data.ByteString (ByteString)
+
+import Control.DeepSeq
+
+import Cardano.Crypto.DSIGN.Class
+import Cardano.Crypto.DSIGN.Ed25519
+import Cardano.Crypto.DSIGN.EcdsaSecp256k1
+import Cardano.Crypto.DSIGN.SchnorrSecp256k1
+import Cardano.Crypto.Hash.Blake2b
+
+import Criterion
+
+import Bench.Crypto.BenchData
+
+{- HLINT ignore "Use camelCase" -}
+
+
+benchmarks :: Benchmark
+benchmarks = bgroup "DSIGN"
+  [ bench_dsign (Proxy :: Proxy Ed25519DSIGN) "Ed25519"
+  , bench_dsign (Proxy :: Proxy EcdsaSecp256k1DSIGN) "EcdsaSecp256k1"
+  , bench_dsign (Proxy :: Proxy SchnorrSecp256k1DSIGN) "SchnorrSecp256k1"
+  ]
+
+bench_dsign :: forall v a
+           . ( DSIGNAlgorithm v
+             , ContextDSIGN v ~ ()
+             , Signable v a
+             , ExampleSignable v a
+             , NFData (SignKeyDSIGN v)
+             , NFData (VerKeyDSIGN v)
+             , NFData (SigDSIGN v)
+             )
+          => Proxy v
+          -> String
+          -> Benchmark
+bench_dsign _ lbl =
```

Comment:
Let's follow the naming scheme of the library and Haskell convention. I know other benchmarks already use this scheme, but I am suggesting we should fix it in all benchmarks
```suggestion
benchDSIGN :: forall v a
           . ( DSIGNAlgorithm v
             , ContextDSIGN v ~ ()
             , Signable v a
             , ExampleSignable v a
             , NFData (SignKeyDSIGN v)
             , NFData (VerKeyDSIGN v)
             , NFData (SigDSIGN v)
             )
          => Proxy v
          -> String
          -> Benchmark
benchDSIGN _ lbl =
```